submarine-sdk/pysubmarine/submarine/proto/SubmarineServerProtocol_pb2.py

```diff
@@ -0,0 +1,1075 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Maybe we should not add the generated files in the git version management. For java, the protocolbuf generated files are located under target folder. So they are not managed by git. I'm not sure about how to work out for python.

Reply:
Let me check other python projects how to handle proto files.